src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/IdeaCollection/Workbench/Workbench.ts

```diff
@@ -145,7 +145,8 @@ export var addProposalButtonDirective = (
 export var registerRoutesFactory = (
     ideaCollection,
     proposalType,
-    proposalVersionType
+    proposalVersionType,
+    hasCommentColumn = true
```

Comment:
It would be much nicer if this could use the processProperties. But I don't see how this could be done in a nice way.